### PR TITLE
Updated both projects to use net6.0

### PR DESCRIPTION
**Pull request by v-michaelmi**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... Please describe: Updating samples to use NET 6.0
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Smoke test the code. Should work same as before.
  * Should also work using the **dotnet cli** using `dotnet run` for each project.

```
```

## What to Check
Verify that the following are valid
* There are no breaking changes

### CODE REVIEW

File: 1. Desktop app calls Web API/TodoListClient/MainWindow.xaml.cs [hunk 1]
```diff
@@ -64,7 +64,7 @@ public MainWindow()
             InitializeComponent();
             _app = PublicClientApplicationBuilder.Create(ClientId)
                 .WithAuthority(Authority)
-                .WithRedirectUri("http://localhost") // needed only for the system browser
+                .WithDefaultRedirectUri()
```
Comment (v-michaelmi) [inline 1. Desktop app calls Web API/TodoListClient/MainWindow.xaml.cs:67]: This defaults to `http://localhost` in .NET Core

File: 2. Web API now calls Microsoft Graph/TodoListService/Properties/launchSettings.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "http://localhost:1040/"
+      "applicationUrl": "https://localhost:44351/"
```
Comment (v-michaelmi) [inline 2. Web API now calls Microsoft Graph/TodoListService/Properties/launchSettings.json:26]: This change is needed for the .NET client to be recognized.

File: 1. Desktop app calls Web API/TodoListService/appsettings.json [hunk 3]
```diff
@@ -10,13 +10,12 @@
     "Domain": "qualified.domain.name", // for instance contoso.onmicrosoft.com. Not used in the ASP.NET core template
     "TenantId": "common" // The tenant's Directory ID (a guid) or 'common' or 'organizations' or 'consumers'
   },
-  "Kestrel": {
-    "Endpoints": {
-      "Http": {
-        "Url": "http://localhost:44351"
-      }
```
Comment (v-michaelmi) [inline 1. Desktop app calls Web API/TodoListService/appsettings.json:17]: This is the setting that was causing the breaks when using .NET 6.0

At first I thought it was unsupported but now I'm seeing some conflicting documentation.

File: 1. Desktop app calls Web API/TodoListClient/TodoListClient.csproj [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 	<PropertyGroup>
 		<OutputType>WinExe</OutputType>
-		<TargetFramework>net5-windows10.0.17763.0</TargetFramework>
+		<TargetFramework>net6.0-windows</TargetFramework>
```
Comment (bgavrilMS) [inline 1. Desktop app calls Web API/TodoListClient/TodoListClient.csproj:5]: Will this work? I think it needs to be net6.0-windows10...

**Review by bgavrilMS — APPROVED** [review]
LGTM. But we should really change the client app to use WAM, not system browser.

**Comment by bgavrilMS** [comment]
Updated the build, hopefully it'll pass now.

**Comment by v-michaelmi** [comment]
Obsolete.

Closing

**Pull request closed without merging**